assets/common/components/deploy_model/spec.yaml

```diff
@@ -39,9 +40,17 @@ inputs:
   # Output of registering component
   registration_details:
     type: uri_file
-    optional: false
+    optional: true
     description: Json file that contains the ID of registered model to be deployed
 
+  model_id:
+    type: string
+    optional: true 
+    description: |
+      Asset ID of the model registered in workspace/registry. Asset ID format looks like below.\
```

Comment:
`\` at end is not needed here. Just make sure to give a space at the end of previous line.

Reply:
Done